bulk_merger.rb

```diff
@@ -17,21 +17,23 @@ def self.approve_unreviewed_pull_requests!
       puts "- '#{pr.title}' (#{pr.html_url}) "
     end
 
-    puts "\nHave you reviewed the changes, and you want to approve all these PRs? [y/N]\n"
-    if STDIN.gets.chomp == "y"
-      puts "OK! Approving away..."
-    else
-      puts "👋"
-      exit 1
-    end
+    unless list
```

Comment:
Replacing this with `return if list` will remove the need to adjust the indentation.

Reply:
:brain: Thanks!